src/Pages/Cart/Cart.tsx

```diff
@@ -65,11 +68,21 @@ const Cart: React.FC = () => {
               {`Total for ${cart.length} items`}
             </h3>
             <div className={checkoutButton}>
-              <Button label="Checkout" onClick={goBack} height="48px" />
+              <Button
+                label="Checkout"
+                onClick={() => setShowModal(true)}
+                height="48px"
+              />
             </div>
           </div>
         </div>
       </div>
+      <Modal
+        title="Thank you for purchase"
+        onClose={() => setShowModal(false)}
+        showModal={showModal}
+        description="Thank you for your purchase. Seriously. There’s no such thing as too much small business support. Please enjoy 20% off your next order with code CODEHUNTERS_TOP"
```

Comment:
let's leave this secondary text:
Please enjoy 20% off your next order with promocode CODEHUNTERS_TOP

Make CODEHUNTERS_TOP bold